scripts/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+declare -a NIXOPS_OPTIONS DEPLOY_FIRST
+NIXOPS_OPTIONS=( -d disciplina --show-trace )
+DEPLOY_FIRST=( witness0 witness-load-balancer )
+SLEEP='3m'
+
+nixops deploy "${NIXOPS_OPTIONS[@]}" --include "${DEPLOY_FIRST[@]}"
+sleep "$SLEEP"
+nixops deploy "${NIXOPS_OPTIONS[@]}" --exclude "${DEPLOY_FIRST[@]}"
```

Comment:
will this staggering work?

Reply:
It did last time I deployed anything. I tried a few intervals and 15m seemed ok, although the chain is now longer.